### DISCUSSION TITLE
Support for subscriptions and recurring payments system

### DISCUSSION THREAD

**Discussion by m-baumgartner** [abpframework/abp#7808] [Ideas]
Hello,

I've seen "Subscription and payment system for the SaaS module." is an item on the roadmap, but i can't find it in here. Is it planned in any upcoming release?

Regards,

**Comment by ebicoglu** [comment]
that's PRO feature if you mention about this one => https://docs.abp.io/en/commercial/latest/road-map
The ABP Framework roadmap is https://docs.abp.io/en/abp/latest/Road-Map

Sorry but that'll be a only in the commercial side.

**Comment by m-baumgartner** [comment]
Got it thanks.
Any idea when this will be brought to ABP Commercial?

**Comment by willignicolas** [comment]
Hi @ebicoglu

We are interested too for this feature in commerciale version. Have you any idea when this is coming out because it's in the roadmap?

Thanks.

**Reply by willignicolas** [reply]
Hello @ebicoglu

Any news for that question ? 

Thanks
Nicolas
